resources/ext.drawioeditor.js

```diff
@@ -239,6 +239,8 @@ DrawioEditor.prototype.save = function(datauri) {
 
     // convert base64 to uint8 array
     datastr = atob(parts[4]);
+	var expr = /"http:\/\/\S*?1999\S*?"/gmi;
```

Comment:
This would also match "http://test.de" attribute="1999". Instead of \S, it's probably better to match [^"]
Also, please check indentation.